test/unit/utils/Serializer.util.test.ts

```diff
@@ -14,14 +14,93 @@ limitations under the License.
 import * as SerializerUtil from "../../../src/utils/Serializer.util";
 
 describe("serializer", () => {
-  it("Object should be serialized to Buffer", () => {
-    const data = SerializerUtil.serializeGrpc({ Hello: "World" });
-    expect(Buffer.compare(data.serializedData, Buffer.from(JSON.stringify({ Hello: "World" })))).toEqual(0);
-    expect(data.contentType).toEqual("application/json");
+  describe("grpc", () => {
+    it("Object should be serialized to Buffer as application/json", () => {
```

Comment:
Just a nit if you have time, these tests should be in a format - "it should ... XYZ", for e.g., `it("should serialize object to buffer as application/json")`.